training/controllers/auth.py

```diff
@@ -1,4 +1,6 @@
 from flask import Blueprint, request, redirect, render_template, url_for, g, session
+from itsdangerous import URLSafeTimedSerializer
+import os
```

Comment:
standard library dependencies import should be at the top, the import order should be:

```
1. standard library imports
2. third party library imports
3. local imports
```

Reply:
Changed!